pkg/middlewares/auth/forward.go

```diff
@@ -66,6 +69,14 @@ func NewForward(ctx context.Context, next http.Handler, config dynamic.ForwardAu
 		fa.client.Transport = tr
 	}
 
+	if config.AuthResponseHeadersRegex != "" {
+		re, err := regexp.Compile(config.AuthResponseHeadersRegex)
+		if err != nil {
+			return nil, fmt.Errorf("failed to compile 'forwardAuth.AuthResponseHeadersRegex' value '%s': %w", config.AuthResponseHeadersRegex, err)
```

Comment:
Thanks for you modifications ;-)
I'm not sure here, i would go this (like in other middlewares)

```suggestion
			return nil, fmt.Errorf("error compiling regular expression %s: %w", config.AuthResponseHeadersRegex, err)
```